### DISCUSSION TITLE
initial release

### DISCUSSION THREAD

**Discussion by binarynoise** [binarynoise/AutomaticAdvancedSettingsExpander#1] [Announcements]
It works on my phone, join the discussion if yours is not supported.

<hr /><em>This discussion was created from the release <a href='https://github.com/binarynoise/AutomaticAdvancedSettingsExpander/releases/tag/'>initial release</a>.</em>

**Comment by Sadyantssr** [comment]
Not working on my phone...xiaomi poco f1

**Reply by binarynoise** [reply]
Not working in the sense of "doesn't do anything" or "crashes something"?

**Reply by binarynoise** [reply]
@Sadyantssr any details?

**Comment by Sadyantssr** [comment]
> It works on my phone, join the discussion if yours is not supported.
> 
> _This discussion was created from the release [initial release](https://github.com/binarynoise/AutomaticAdvancedSettingsExpander/releases/tag/)._

 Which advanced settings are you talking about?

**Reply by binarynoise** [reply]
| before | after |
|--------|-------|
|![before](https://user-images.githubusercontent.com/50302352/156211967-6d72f9c2-4185-4e75-8e88-0dc9b282945e.png)|![after](https://user-images.githubusercontent.com/50302352/156211962-ced38048-4970-4fa4-bdb0-51fcba2b4f6a.png)|

**Reply by binarynoise** [reply]
I think, I'll include that in the Readme

**Reply by binarynoise** [reply]
@Sadyantssr does this answer your question

**Comment by arubamon** [comment]
Does not work on my ROM, [𝑹𝑶𝑴][12.1]𝑨𝒘𝒂𝒌𝒆𝒏 𝑶𝑺 [𝑶𝑵𝑬𝑷𝑳𝑼𝑺 6-6𝑻] WITH-UPDATER 12.1L 𝒃𝒚 𝒐𝒑𝒕𝒊𝒎𝒖𝒔.  Hoping you can get it to work on Android 12.1, because this tweak is very helpful.  Thanks in advance

**Reply by binarynoise** [reply]
@arubamon 
I'll have a look when I am back from 
holidays, although it works on my lineage os install with android 12.
Can you link the xda (?) Thread where you get your rom?

**Reply by arubamon** [reply]
@binarynoise, I am on [Awaken-2.9-Aero-fajita-UNOFFICIAL-Gapps-1111-20220715.zip](https://sourceforge.net/projects/awakenos-op6-t/files/fajita/Awaken-2.9-Aero-fajita-UNOFFICIAL-Gapps-1111-20220715.zip/download), which can be found at https://sourceforge.net/projects/awakenos-op6-t/files/fajita.  The thread on XDA is https://forum.xda-developers.com/t/rom-12-1-awaken-os-oneplus-6-6t-with-updater-12-1l-by-optimus-11-07-2022-july.4408737.  Thank you again

**Reply by binarynoise** [reply]
It looks like Awaken is based on AOSP and uses the same codebase. Therefore, it should just work.
Do you have any messages in the LSPosed logs or logcat? Can you share them?
